tests/service/data/samename/tempet.fsproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="FSharp.NET.Sdk;Microsoft.NET.Sdk">
```

Comment:
I'm not sure about ``tempet.fsproj``, but the tests in ``tests/service/ProjectAnalysisTests.fs`` are, I think, also being run against FSharp.LanguageService.Compiler.dll (the locally built compiler service DLL)